platforms/hyperledger-indy/scripts/indy_nym_txn/generate_txn.txt

```diff
@@ -0,0 +1,15 @@
+wallet create mywallet key=pass12345
+wallet open mywallet key=pass12345
+wallet list
+did import ./admindid.txt
+did import ./dids.txt
+did list
+
+did use Th7MpTaRZVRYnPiabds81Y
+
+pool create mysandbox gen_txn_file=./genesis_pool_config
+pool connect mysandbox
+pool list
+
+ledger nym did=AAAMpTaRZVRYnPiabds81Y verkey=8QhFxKxyaFsJy4CyxeYX34dFH8oWqyBv1P4HLQCsoeLy role=TRUST_ANCHOR
```

Comment:
How can we determine the suceess / failure of this command?

Reply:
If the txn is successful, it displays the success message in the console that NYM has been received along with the metadata.
If it fails (like wrong verkey or verkey modification by stewerd) error msg is displayed that NYM has  been rejected.

Reply:
and how are we able to automate this check?

Reply:
When the script runs it open and external terminal and displays the corresponding msg if successful or failed

Reply:
and how would you automate the check whether the script encountered an error?